cmd/operator/main.go

```diff
@@ -98,6 +98,8 @@ func checkPrerequisites(
 const (
 	defaultReloaderCPU    = "10m"
 	defaultReloaderMemory = "50Mi"
+
+	defaultMemlimitRatio = 0.9
```

Comment:
```suggestion
	defaultMemlimitRatio = 0.0
```

Reply:
Got it. Setting defaultMemlimitRatio default to 0.0 (disabled)